templates/raspberry-pi/desktop.html

```diff
@@ -0,0 +1,159 @@
+{% extends "raspberry-pi/_base_raspberry-pi.html" %}
+
+{% block title %}Raspberry Pi with Ubuntu CLI
+{% endblock %}
+
+{% block meta_description %}The Raspberry Pi is an ARM computer for everybody. Perfect for teaching, coding, surfing the web or simply as a desktop for anyone, anywhere. The Raspberry Pi with Ubuntu is a gateway to the world of open source invention.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/14WEwUuwaKGMaR5FlsJJZttkAn0iCqUuXhRQLMWurpQ4/edit{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-topped is-dark">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-6">
+      <h1>Ubuntu Desktop<br>for anyone</h1>
+      <p>
+        Everyone should have a desktop. Whether you’re surfing the web, developing an app, or working in the cloud, you can do it all from the Raspberry Pi 4.
+      </p>
+      <p>
+        <a href="/download/raspberry-pi">
+          Download Ubuntu Server for Raspberry Pi&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+    <div class="col-6 u-align--right">
```

Comment:
I normally put the `u-hide--small` here, not sure it matters

Reply:
@albertkol  also, I have another PR for the footer, so ignore that comment in the issue